### DISCUSSION TITLE
Video Editing Resource

### DISCUSSION THREAD

**Discussion by beatricehc** [IBEXImagingCommunity/ibex_imaging_knowledge_base#198] [Show and tell]
Hello everyone!
I wanted to share the web-based video editing software ClipChamp that makes compiling clips and editing videos extremely easy. It is a free application and does not require downloading any software as it is entirely online. I hope someone finds this useful!

https://clipchamp.com/en/

**Comment by radtkea** [comment]
AWESOME! Thank you so much @beatricehc for sharing this resource! We have been looking for a free, easy to use, high quality solution for video editing. There are tons of helpful tutorials on the ClipChamp [YouTube Channel](https://www.youtube.com/@clipchampcom). Other content creators provide a quick [introduction](https://www.youtube.com/watch?v=V8DvKd2elak) to ClipChamp and detail how to do [voiceovers](https://www.youtube.com/watch?v=Lijy_SyPxcI) including from text to speech with an AI voice! ClipChamp even offers [AI transcription to text](https://www.youtube.com/watch?v=pbko4WBzx2o) for free. 

Thank you so much! This will help many create [tutorial videos](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/videos.html) and content for publications.

**Comment by DasEFX130** [comment]
That was really interesting. I logged onto it through my work account and it turned out we use this software from my company. I had no clue! Thanks for the resource!

**Reply by radtkea** [reply]
Hooray!!! Thank you so much for sharing:)
